BrainPortal/app/views/portal/_available_datasets_table.html.erb

```diff
@@ -41,7 +41,14 @@
     end
   %>
   <tr>
-    <td><%= overlay_description((name + "\n" + description).strip) %></td>
+    <td>
```

Comment:
Why is this done with an explicit construction of the H1, HR and P elements ? This is cumbersome and will result in empty overlays when the description is empty.

Look at the original code, it was invoking a HELPER. This what what helpers are for: they can make decision about what to render, make adjustements, and even decide not to render anything at all.

With a help such as closable_overlay_content_link() (name I just invented, adjust it please) you could have instead:

```html
<td><%= closable_overlay_content_link(name, description) %></td>
```

which would make all these decisions and not leave empty P elements if there are no descriptions.